lib/node.js

```diff
@@ -675,6 +676,7 @@ declare module "events" {
     static EventEmitter: typeof EventEmitter;
   }
 
+  declare export var EventEmitter: typeof EventEmitter;
```

Comment:
No idea why this is needed, given that `EventEmitter` is already declared above. @jeffmo please have a look.